File: Grocerly.Hybrid/Grocerly.Hybrid/Grocerly.Hybrid/Models/Shoppinglist.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Grocerly.Hybrid.Models
{
    public class ShoppingList
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public Status Status { get; set; }
    }
}

File: Grocerly.Android/app/src/main/java/grocerly/fhict/com/grocerly/models/ShoppingList.java
package grocerly.fhict.com.grocerly.models;

import java.util.List;

public class ShoppingList {
    private List<Product> products;
    private String id;
    private String name;

    public List<Product> getProducts() {
        return products;
    }

    public void setProducts(List<Product> products) {
        this.products = products;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }
}
